File: Fundamentals of C programming/step07/7.3/7.3.02.c
/*

Пирамида.
Вывести на экран пирамиду из чисел (см. пример входных данных).

Входные данные:
Одно целое число NN.

Выходные данные:
Пирамида из натуральных чисел высоты NN.

Sample Input 1:

5

Sample Output 1:

1 
1 2 
1 2 3 
1 2 3 4 
1 2 3 4 5 

Sample Input 2:

1

Sample Output 2:

1 

*/

#include <stdio.h>

int main() {
    int n, i, j;
    scanf("%d", &n);
    for (i = 1; i <= n; i++){
        for (j = 1; j <= i; j++) {
            printf("%d ", j) ;
        }
        printf("\n") ;
    }
    return 0;
}

File: Indie Python Programming Course/step07/7.03/7.3.02.py
'''

Давайте считать человека подростком, если его возраст находится в пределах от 12 до 17 лет включительно. 
Напишите функцию is_person_teenager , которая помогает по возрасту определить является ли человек подростком или нет.

Функция is_person_teenager принимает на вход возраст человека и возвращает True или False

Нужно написать только определение функций is_person_teenager 

Sample Input 1:

8

Sample Output 1:

False

Sample Input 2:

15

Sample Output 2:

True

'''

def is_person_teenager(n):
    if n <= 17 and n >=12:
        return True
    return False
